### ISSUE TITLE
Crashes on opening a .js file

### ISSUE THREAD

**Issue by tusharmath** [hax/refactor#7]
steps to reproduce
1. Install js-refactor
2. Instal refactor
3. Open a .js file

**Atom Version**: 1.0.7
**System**: Mac OS X 10.10.4
**Thrown From**: [js-refactor](https://github.com/hax/js-refactor) package, v0.4.2
### Stack Trace

Uncaught TypeError: Cannot read property '18' of undefined

```
At /Users/tushar.mathur/.atom/packages/js-refactor/lib/ripper.coffee:49

TypeError: Cannot read property '18' of undefined
  at Ripper.module.exports.Ripper.find (/Users/tushar.mathur/.atom/packages/js-refactor/lib/ripper.coffee:49:21)
  at Watcher.module.exports.Watcher.createReferences (/Users/tushar.mathur/.atom/packages/refactor/lib/watcher.coffee:135:22)
  at Watcher.module.exports.Watcher.onParseEnd (/Users/tushar.mathur/.atom/packages/refactor/lib/watcher.coffee:108:8)
  at Watcher.onParseEnd (/Users/tushar.mathur/.atom/packages/refactor/lib/watcher.coffee:1:1)
  at Watcher.module.exports.Watcher.parse (/Users/tushar.mathur/.atom/packages/refactor/lib/watcher.coffee:102:8)
  at /Users/tushar.mathur/.atom/packages/refactor/lib/watcher.coffee:1:1

```
### Commands

```
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
file-icons, v1.6.7
js-refactor, v0.4.2
language-scala, v1.1.3
linter-scalac, v1.0.2
minimap, v4.12.3
react, v0.12.6
refactor, v0.5.1

# Dev
No dev packages
```

**Comment by TBotV63** [comment]
Could you change the title to be more descriptive please?

**Comment by hax** [comment]
Could you paste the js code that cause crash?

**Comment by tusharmath** [comment]
Happens in any js file.

**Comment by hax** [comment]
I just published a new version (0.6.0), could you update to the latest version and check the status?

**Comment by tusharmath** [comment]
Fixed. Sorry for a delayed response. I had given up on Atom and started using webstorm.
